src/alias.cpp

```diff
@@ -2445,11 +2445,11 @@ UniValue aliasinfo(const JSONRPCRequest& request) {
 	vector<unsigned char> vchAlias = vchFromValue(params[0]);
 	ToLowerCase(vchAlias);
 	CAliasIndex txPos;
-	if (!paliasdb || !paliasdb->ReadAlias(vchAlias, txPos))
+	if (!paliasdb)
```

Comment:
Should be a unit test for both conditions, one for each?

Reply:
i don't think the unit test checks for the correct error message